openapi/find-my-trn-api.yml

```diff
@@ -0,0 +1,291 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: teacher-identity
+servers:
+  - url: 'http://localhost:8080'
+paths:
+  /trn-email-match:
+    get:
+      summary: 'teacher endpoint: returns trn details given an email'
+      parameters:
+        - name: email
+          in: query
+          description: email of person
+          required: true
+          schema:
+            type: string         
+      responses:
+        '200':
+          description: 'Ok'
+          headers:
+            x-next:
```

Comment:
Do we need paging here?

Reply:
no, good point. Hangover from my API OAS template I use. Will remove.